projects/ngx-quill/package.json

```diff
@@ -31,9 +31,9 @@
   ],
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=13.0.0",
+    "@angular/core": "^14.0.0",
     "quill": "^1.3.7",
-    "rxjs": "^6.5.3 || ^7.4.0"
+    "rxjs": "^7.0.0"
```

Comment:
Just wanted to point out that because of this version bump, this PR is a breaking change 
- 17.0.0 could be used in an existing angular app with rxjs v6.
- 17.1.0 cannot be used in the same existing app due to the dep.

Anyway, thanks for maintaining the project!